app/code/Magento/Sales/Model/Order/Invoice/Total/Grand.php

```diff
@@ -8,17 +8,14 @@
 class Grand extends AbstractTotal
 {
     /**
+     * Collect invoice grand total
+     *
      * @param \Magento\Sales\Model\Order\Invoice $invoice
      * @return $this
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
     public function collect(\Magento\Sales\Model\Order\Invoice $invoice)
     {
-        /**
-         * Check order grand total and invoice amounts
-         */
-        if ($invoice->isLast()) {
-            //
```

Comment:
Same concern as https://github.com/magento/magento2/pull/14106/files#r174779579 - prior to removal we should understand the intention of code author.

Reply:
@orlangur 
![image](https://user-images.githubusercontent.com/5318512/37466689-144c5e4a-2867-11e8-9c7f-244b2789645d.png)
As I can see this code was added to the repo with the first push :)